pkgs/tools/filesystems/zxfer/default.nix

```
@@ -0,0 +1,41 @@
+{ lib, stdenv, fetchFromGitHub
+, coreutils, gnused, gawk, gnugrep
+, openssh, rsync
+, gzip }:
+
+stdenv.mkDerivation rec {
+  pname = "zxfer";
+  version = "1.1.7";
+
+  src = fetchFromGitHub {
+    sha256 = "11SQJcD3GqPYBIgaycyKkc62/diVKPuuj2Or97j+NZY=";
+    rev = "v${version}";
+    repo = "zxfer";
+    owner = "allanjude";
+  };
+
+  buildInputs = [ coreutils gawk gnused gnugrep openssh rsync ];
+  nativebuildInputs = [ gzip ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp zxfer $out/bin/
+
+    mkdir -p $out/share/man/man8/
+    cp zxfer.8 $out/share/man/man8/
+  '';
+
+  meta = with lib; {
+    inherit version;
+    inherit (src.meta) homepage;
```

Adding the URL allows one to visit the homepage from the editor.

```suggestion
    homepage = "https://github.com/allanjude/zxfer";
```